packages/governance/src/contractGovernance/governParam.js

```diff
@@ -51,13 +55,22 @@ const validateParamChangeQuestion = details => {
   );
 };
 
-/** @type {AssertBallotConcernsQuestion} */
-const assertBallotConcernsQuestion = (paramName, questionDetails) => {
+/** @type {AssertBallotConcernsParam} */
+const assertBallotConcernsParam = (paramSpec, questionDetails) => {
+  const { key, parameterName } = paramSpec;
+  /** @type {ParamChangeIssue} */
+  // @ts-ignore cast
+  const issue = questionDetails.issue;
+  assert(issue, X`must be a param change issue`);
+
+  const foundName = issue.changes.find(c => c.parameterName === parameterName);
```

Comment:
if `changes` were a map this would be O(1) lookup

Reply:
I refactored from a list of changes to a record.